debian/control

```diff
@@ -0,0 +1,19 @@
+Source: golang-logrus
+Section: devel
+Priority: extra
+Maintainer: Jessie Frazelle <jess@docker.com>
+Uploaders: Docker Packaging Team <docker-maint@lists.alioth.debian.org>
+Build-Depends: debhelper (>= 9), dh-golang, golang-go, golang-go.net-dev
+Standards-Version: 3.9.5
```

Comment:
We might as well bump this to `3.9.6` while we're here too (`W: golang-logrus source: out-of-date-standards-version 3.9.5 (current is 3.9.6)`), but do be careful about getting too crazy with going "lintian clean" (which is part of the reason why I _always_ run lintian with the [`-EvIL+pedantic` flags](https://nthykier.wordpress.com/2012/02/23/some-sponsors-are-evil-and-pedantic/) -- that way there's pretty much always at least one that I literally can't fix so I don't get crazy about forcing myself to fix them all, especially since some of them really needn't be fixed).